#167486077 Add a migration shell script as an entry point to run migrations

ssewilliam opened the pull request:
#### What does this PR do?
- Add a shell script as an entry point to run migrations for every image that is to be built

#### Description of Task to be completed?
- Add a shell script as an entry point to run migrations for every image that is to be built

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167486077](https://www.pivotaltracker.com/story/show/167486077)

valmayaki commented:
You actually did more than I what I think is needed. This is another direction, test it and be sure it works. The only concern I have is that it might override the previous template that is being used, confirm that it doesn't. 
The easiest method would have been to create a shell script which would have to replace the default command, this would have included the migration script ` yarn migrate` and ` node  ./build/index.js` e.g . `yarn migrate && node  ./build/index.js`  therefore any time the container is run migration would run first before the node server.

ssewilliam committed:
ch(migrationScript): Add a shell script to run migrations before starting the application
- Add a shell script to run migrations before starting the application

[Starts #167486077]

valmayaki commented on scripts/entrypoint.sh:
```diff
@@ -0,0 +1,13 @@
+printf "\n======================================\n\n"
+node ./build/index.js
+
+exit 0
```
remove the exit 0; by default `node ./build/index.js` will continuously run without reaching the `exit 0` statement

ssewilliam replied:
Done

valmayaki commented:
looks good

Gidraff reviewed (commented):
LGTM
